backend/app/migrations/0011_merge_document_gender_branches.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.1.5 on 2021-06-24 18:55
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0004_auto_20210623_1831'),
+        ('app', '0010_pronounseries_fields_blank_false'),
+    ]
+
+    operations = [
+    ]
```

Comment:
Two considerations:

1. I'm not sure what this migration is doing, but it looks accidental.
2. Since the main branch doesn't contain any reference to the models that you created and then deleted in this branch, we don't necessarily need to include these migrations in the main branch. I think a more succinct approach would be to delete all of the migrations you've created here and rerun `makemigrations`. This will compile all changes in the `models.py` file into a single migration. If we do that, we'd want to squash merge your branch into main, so there'd only be a single commit and this new single migration file representing your work.

Reply:
That last migration file was created when I merged the `main` branch in before submitting the PR: the two files in the `dependencies` list are the ends of the two branches of files stemming from the `0002_document.py` migration. DJango had thrown an error describing a merge conflict and prompted to run `py manage.py makemigrations --merge`, which created that file. After running `migrate`, everything seemed fine.

I don't really know what that migration file does (since the branches don't at all depend on each other), but it seemed to resolve that error.

With deleting the migrations file and running `makemigrations`, does that mean rolling back to `0002_document` and deleting all migrations after that, and then running `makemigrations`?

Reply:
Going the deletion route would mean removing all migrations from this PR that aren't in the `main` branch (it looks like there's 4 atm), then rerunning `makemigrations`.

Reply:
Ok, I ended up squashing the branch of migration files into one that depended on `0002_document.py`. The `CommandError` I receive from the merge conflict with the branch of migrations concerning the `Document` model persists without the extra merge migration file. While that migration does not carry out any operations, I believe the ordering of migrations in the `dependencies` list is vital for resolving the order of execution for the two branches of migrations.